content/applications/marketing/email_marketing/mailing_lists.rst

```diff
@@ -9,128 +9,212 @@ feedback, and more.
-which contacts can be added to this specific list by clicking :guilabel:`Create`, and adding contact
-information directly on a separate contact detail form.
+Doing so reveals a separate :guilabel:`Mailing List Contacts` page for that specific mailing list,
+where contacts can be created or imported, and then added to the specific mailing list.
 
-Or, while in the :menuselection:`Email Marketing` application, navigate to :menuselection:`Mailing
-Lists --> Mailing List Contacts`. Doing so reveals a separate page with all the mailing list
-contacts in the database.
+This same page can also be accessed by clicking the desired mailing list from the :guilabel:`Mailing
+Lists` page, and then clicking the :guilabel:`Recipients` smart button on the mailing list form.
 
-From here, click :guilabel:`Create`, and add a contact with the same previous steps. Or, click the
-:guilabel:`Import` icon (to the right of the :guilabel:`Create` button) to import contacts into the
-database.
+Doing so *also* reveals a separate :guilabel:`Mailing List Contacts` page for that specific mailing
+list, where contacts can be created or imported, and then added to the specific mailing list.
 
-Once contacts are in the database, click into the desired contact's detail form, and add the
-preferred mailing list in the :guilabel:`Mailing List` tab (at the bottom of the contact detail
-form), by clicking :guilabel:`Add a line`, and selecting the desired mailing list. Multiple mailing
-lists can be added to a single contact's detail form.
+Contacts can also be directly imported to a specific mailing list from the :guilabel:`Mailing Lists`
+page, by clicking :guilabel:`Import Contacts` to the far-right of the desired mailing list.
 
-.. image:: mailing_lists/contact-form-mailing-list-tab.png
+.. image:: mailing_lists/import-contacts-button.png
    :align: center
-   :alt: View of a contact detail form with mailing list tab in Odoo Email Marketing.
+   :alt: The import contacts button of a mailing list line in Odoo Email Marketing.
 
-Create new mailing list from contact detail form
-------------------------------------------------
+Doing so reveals an :guilabel:`Import Mailing Contacts` pop-up form.
 
-To create a mailing list from a contact detail form, click :guilabel:`Add a line`, and type in the
-name of a new mailing list in the empty field that appears. Then, select either :guilabel:`Create`
-or :guilabel:`Create and Edit...`.
+.. image:: mailing_lists/import-mailing-contacts-popup.png
+   :align: center
+   :alt: The import mailing contacts pop-up form that appears in Odoo Email Marketing.
+
+Here, the desired mailing list is auto-populated in the :guilabel:`Import contacts in` field.
+Beneath that, write or paste email addresses in the :guilabel:`Contact List` field.
+
+The option to import a country, company name, and more is available, via the :guilabel:`Upload a
+file` link at the bottom of the pop-up form.

```

Comment:
Is there a upload template provided in the database? Can we add it here?
Even though this appears a secondary action, file uploads are actually the primary way of building email lists, generally speaking.